### ISSUE TITLE
Cookies overload (no deletion/replacing)

### ISSUE THREAD

**Issue by lanodan** [OtterBrowser/otter-browser#1318]
After a while (a day or more), I have a 400 Error because otter sends too much cookies and it doesn’t seems to delete itself and/or overwrite so when cookies values are different/the same it keeps storing it and sending a huge load of cookies.

Notes:
* I filter cookies(and javascript) per website
* I use QtWebkit
* Versions
  * Otter release (0.9.12) and git HEAD(6d24316)
  * Qt 5.6.2
  * uname -a : Linux OpalEssence 4.8.17-hardened-r2 #5 SMP Thu Feb 16 20:32:51 UTC 2017 x86_64 Intel(R) Pentium(R) CPU P6000 @ 1.87GHz GenuineIntel GNU/Linux

**Comment by ersi-dnd** [comment]
Yes, this problem is back for me too, after just a few clicks when cookies enabled and logged in somewhere http://thedndsanctuary.eu/index.php?topic=1373.0
